src/content/Effects.js

```diff
@@ -226,7 +206,7 @@ Possible fixes:
                     <tr>
```

Comment:
I would remove the entire line.

Reply:
I'm not sure which line you are refering to. Is it the in the table at the bottom of the page?

Reply:
Yes, the row in the table.

Reply:
done